src/ripple/app/main/Application.cpp

```diff
@@ -404,7 +404,9 @@ class ApplicationImp
         //
         , m_jobQueue (std::make_unique<JobQueue>(
             m_collectorManager->group ("jobq"), m_nodeStoreScheduler,
-            logs_->journal("JobQueue"), *logs_))
+            logs_->journal("JobQueue"), *logs_,
+            config_->exists (SECTION_VALIDATOR_TOKEN) ||
```

Comment:
I think this is correct, but paging @wilsonianb to be sure it is forward compatible with dynamic UNL.